src/frontend-scripts/components/section-main/Profile.jsx

```diff
@@ -322,7 +328,20 @@ class ProfileWrapper extends React.Component {
 								<span>{gamesUntilRainbow}</span>
 							</div>
 						)}
+						{!isNaN(gamesUntilRainbow) && (
+							<div>
+								<span>
+									<strong>
+										<em>Games Until Rainbow (Season): </em>
+									</strong>
+								</span>
+								<span>{gamesUntilRainbowSeason}</span>
+							</div>
+						)}
 						{profile.lastConnectedIP && <p>Last connected IP: {profile.lastConnectedIP}</p>}
+						{profile.signupIP && <p>Signup IP: {profile.signupIP}</p>}
+						{!profile.staffDisableVisibleElo && <p>ELO (Seasonal): {profile.eloSeason}</p>}

```

Comment:
This is somewhat redundant, we can just hide the elements if `profile.eloSeason` is falsy, and handle the hiding from backend.